packages/grid/_modules_/grid/components/menu/columnMenu/SortMenuItems.tsx

```diff
@@ -0,0 +1,56 @@
+import { MenuItem } from '@material-ui/core';
+import * as React from 'react';
+import { useGridSelector } from '../../../hooks/features/core/useGridSelector';
+import { sortModelSelector } from '../../../hooks/features/sorting/sortingSelector';
+import { ColDef } from '../../../models/colDef/colDef';
+import { SortDirection } from '../../../models/sortModel';
+import { ApiContext } from '../../api-context';
+
+export interface SortMenuItemsProps {
+  column: ColDef;
+  onClick: (event: React.MouseEvent<any>) => void;
+}
+
+export const SortMenuItems: React.FC<SortMenuItemsProps> = ({ column, onClick }) => {
+  const apiRef = React.useContext(ApiContext);
+  const sortModel = useGridSelector(apiRef, sortModelSelector);
+
+  const sortDirection = React.useMemo(() => {
+    if (!column) {
+      return null;
+    }
+    const sortItem = sortModel.find((item) => item.field === column.field);
+    return sortItem?.sort;
+  }, [column, sortModel]);
+
+  const onSortMenuItemClick = React.useCallback(
```

Comment:
IMHO, we shouldn't have a usecallback here

https://kentcdodds.com/blog/usememo-and-usecallback